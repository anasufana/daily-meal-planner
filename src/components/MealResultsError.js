import React from 'react';
import { Link } from 'react-router-dom';

const MealResultsError = () => (
  <h2 className="meal-results-error">
    Ooops! We can&apos;t seem to find a meal plan with theese requirements.<br />
    <Link to="/" href="/">Try again?</Link>
  </h2>
);

export default MealResultsError;
