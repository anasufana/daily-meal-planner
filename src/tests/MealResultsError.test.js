/* global it, expect, jest */

import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import MealResultsError from '../components/MealResultsError';

it('renders correctly', () => {
  const tree = renderer
    .create(<Router><MealResultsError
    /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
