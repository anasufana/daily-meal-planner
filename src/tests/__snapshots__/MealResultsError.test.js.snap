// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h2
  className="meal-results-error"
>
  Ooops! We can't seem to find a meal plan with theese requirements.
  <br />
  <a
    href="/"
    onClick={[Function]}
  >
    Try again?
  </a>
</h2>
`;
